docs/howto/manage-charms.rst

```diff
@@ -250,6 +251,34 @@ that at present this must be a Discourse page. E.g.,
 
     See more: :ref:`charmcraft-yaml-key-links`
 
+The Description tab can contain additional documentation, such as tutorials or 
+how-to guides.
+
+.. note::
+   A charm's documentation should focus on the charm itself.
+   For workload-specific or Juju-related content, link to the appropriate upstream 
+   documentation.
+
+A smaller charm can have a single-page documentation for its description. 
+A bigger charm, that needs a multi-page documentation, can have either
```

Comment:
```suggestion
A smaller charm can have a single-page document for its description. 
A bigger charm, that needs multi-page documentation, can have either
```

Grammar-related.

Reply:
Yes, we can delete the articles, but I don't think the other word has the same meaning. We really want to talk about charm's documentation that fits into a single web page.

```suggestion
A smaller charm can have single-page documentation for its description. 
A bigger charm, that needs multi-page documentation, can have either
```